src/Sylius/Bundle/ApiBundle/CommandHandler/SendContactRequestHandler.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\CommandHandler;
+
+use Sylius\Bundle\ApiBundle\Command\SendContactRequest;
+use Sylius\Bundle\CoreBundle\Mailer\Emails;
+use Sylius\Component\Channel\Repository\ChannelRepositoryInterface;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Mailer\Sender\SenderInterface;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+use Webmozart\Assert\Assert;
+
+/** experimental */
+final class SendContactRequestHandler implements MessageHandlerInterface
+{
+    const CONTACT_REQUEST_API = 'contact_request_api';
```

Comment:
I want to avoid "magic strings" in this class, so in the next lines I've used this const, maybe a separate interface with this const will be a better idea, but for now, we have only one email in apiBunlde,

Reply:
But it wasn't used anywhere